src/PageContainer.js

```diff
@@ -2,6 +2,8 @@ import React, {Component} from 'react';
 import { PageContent } from './Enums.js';
 import NavBar from './PageLayout/Navigation/NavBar.js';
 import PageHeader from './PageLayout/PageHeader.js';
+import logo from './icons/temp-logo.svg'
+// import RecurringPickupRequest from './PageContent/RequestPickup/RecurringPickupRequest'
```

Comment:
is this comment intentional?

Reply:
yeah its cause recurring pickup doesn't exist yet in master so I commented it out so it wouldn't crash. I could deleted it too tho